pipeline-model-definition/src/test/java/org/jenkinsci/plugins/pipeline/modeldefinition/actions/RestartDeclarativePipelineActionTest.java

```diff
@@ -179,9 +179,17 @@ public void simpleRestart() throws Exception {
         HtmlForm form = page.getFormByName("restart");
         HtmlTextArea text = form.getTextAreaByName("_.stageName");
         text.setText("restart");
-        HtmlPage redirect = j.submit(form);
-
-        assertEquals(p.getAbsoluteUrl(), redirect.getUrl().toString());
+        try {
+            HtmlPage redirect = j.submit(form);
+
+            assertEquals(p.getAbsoluteUrl(), redirect.getUrl().toString());
+        } catch (Exception we) {
+            // Annoying 404 shows up for http://localhost:60117/jenkins/static/05fb3e5f/plugin/pipeline-stage-view/jsmodules/stageview.js
```

Comment:
@svanoort - So for some reason calling `j.submit(form)` results in a 404 for the above URL (port and `static/.../` path variable, of course). It doesn't actually break the submission, it just throws an exception. I'm stuck with stage view getting sucked in via jira-steps's dependency on workflow-aggregator (siiiiiigh) because I need to be able to test the dementedness jira-steps 1.3.0 did with metasteps, so I'm not sure if there's a way around this. Any ideas? If not, I'll probably sadly remove the jira-steps test and dependency.